Library/Homebrew/formula_specialties.rb

```diff
@@ -18,12 +18,20 @@ def self.url(val)
 # This formula serves as the base class for several very similar
 # formulae for Amazon Web Services related tools.
 class AmazonWebServicesFormula < Formula
+
+  depends_on :java
```

Comment:
So should I leave this as it is, Or ask every downstream formula adding `depends_on :java` them own?